tools/mo/openvino/tools/mo/front/freeze_placeholder_value.py

```diff
@@ -48,6 +48,7 @@ def replace_sub_graph(self, graph: Graph, match: dict):
             try:
                 if data_type != np.bool:
                     value = mo_array(string_value, dtype=data_type)
+                # TODO: investigate why bool is allowed only for TF
```

Comment:
investigate why boolean type is allowed only for TensorFlow only

Reply:
done